### DISCUSSION TITLE
Spawned droids in upperhospital are mute

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#86] [Q&A]
I'm trying to spawn a droid in mission 5, upperhospital entry where you enter the level from the elevator.
For some reason it's missing it's creepy voices.

All droids I add in mission5 are missing their vocalizations.   I'm obviously missing something.
-
Spawn npc/android.npc. 
position coordinates.
spawn_on_reset flag.
Npc_alert = 2 
-
That makes the droid hostile on sight, but he doesn't talk.   It's creepier to have it talk when chasing you.

I was looking at what JeffCat did in Kitty's nightmare. He spawned a droid in the morgue area of M6 inside the center room.
It seems to have everything I've added and only thing extra is an integer switch npc_spawn = 1.
No child nor parent items on the droid entity nor npc alert.  
 
 I reproduced the droid from an unmodified mission 6 file and the droid spawned as expected, but this one had his voice working.
 So what is the difference between M5 and M6 ?
 I guess I'm missing a prerequesite setup for the droid voice sound bank?
 Any help will be appreciated.

**Comment by MattFiler** [comment]
Soundbanks are loaded through `SoundLoadBank` nodes - I suspect the level in question doesn't have the Joe soundbanks loaded if there are none usually present in the level. You might need to trigger this yourself :)

**Reply by ghramsey** [reply]
Thank you.    Yes. That's what I've been trying to trigger.  There's a variablestring function named Android that has sound_bank refrerenced to it missing.   In all maps with droids there are similar entries however, adding this function didn't yield any results.  Somehow, I'm missing the trigger to enable the functions including adding these entries through the upperdecksectionA where I've spawned the droids.  No effect.

I may just leave them mute for now.   As someone pointed out this is becoming a rabbit hole.

**Reply by ghramsey** [reply]
I see something different in the soundbank. It says "trigger via pin". 
"Pin" seems to be a logic gate.   I cannot find where that logic gate is set though.
